### PR TITLE
Directory structure for Kraken future API

### PR DESCRIPTION
**Pull request by rewong03**
### Description
Minor directory/client scaffolding so people can work on the private/public future apis at the same time

### Testing
N/A

### Impact
N/A

### Other
N/A

### CODE REVIEW

File: src/pysrc/adapters/kraken/future/kraken_future_client.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class KrakenFutureClient:
+    def __init__(
```
Comment (echavemann) [inline src/pysrc/adapters/kraken/future/kraken_future_client.py:2]: id change this signature to take some bool: use_testnet/use_live and then ternary the base_url

Comment (rewong03) [reply src/pysrc/adapters/kraken/future/kraken_future_client.py:2]: ? sounds more complicated then just changing the base_url on instantiation

Comment (echavemann) [reply src/pysrc/adapters/kraken/future/kraken_future_client.py:2]: so much work to pass in the URLs lol

Comment (echavemann) [reply src/pysrc/adapters/kraken/future/kraken_future_client.py:2]: or to import them

Comment (rewong03) [reply src/pysrc/adapters/kraken/future/kraken_future_client.py]: done in https://github.com/northwesternfintech/desk-monorepo/pull/30/commits/fd051c66be2024126241da3d872fb932f18077cb

**Review by echavemann — APPROVED** [review]
chk

**Pull request merged**